prober/tcp.go

```diff
@@ -98,6 +98,13 @@ func ProbeTCP(ctx context.Context, target string, module config.Module, registry
 		Name: "probe_ssl_last_chain_expiry_timestamp_seconds",
 		Help: "Returns last SSL chain expiry in unixtime",
 	})
+	probeSSLFingerprint := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "probe_ssl_fingerprint_info",
```

Comment:
This should mention that it's the leaf. Also you generally want to avoid an info metric per label name, so I'd not put fingerprint in the name.

Reply:
@brian-brazil thanks for the hint on the first part, going to change it. Could you please elaborate a bit more what you mean by "info metric per label name" as I'm unable to follow you?

Reply:
Don't put the word fingerprint into the metric name, as that prevents us putting other leaf cert information in here too.

Reply:
Okay got you. Going to change it to `probe_ssl_leaf_info` or do you have a better idea?

Reply:
probe_ssl_last_chain_info would be consistent with other metrics.

Reply:
Is implemeted in [84d73fc](https://github.com/prometheus/blackbox_exporter/pull/678/commits/84d73fcf5ac1637711901d1a70036a410bae5972).